tests/misc_test.go

```diff
@@ -623,3 +623,15 @@ func TestTypeConversion(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestSliceOutOfRange(t *testing.T) {
+	defer func() {
+		if err := recover(); err == nil || !strings.Contains(err.(error).Error(), "slice bounds out of range") {
+			t.Fail()
+		}
+	}()
+
+	a := make([]byte, 4096)
+	b := a[8192:]
```

Comment:
There's no need to use numbers so large, the same issue would happen with `make([]byte, 4096)` and `a[8:]`, right? Maybe it's better to use those smaller numbers to make the test focus more on the underlying issue and less resource intensive.

Reply:
I don't have a strong opinion on it :-) Done.